tensorflow/core/grappler/optimizers/auto_mixed_precision.h

```diff
@@ -24,6 +24,12 @@ namespace grappler {
 
 enum class AutoMixedPrecisionMode { CUDA, MKL };
 
+// Note: This is primarily used by the tf.experimental.tensorrt. Converter class
+// to use mixed precision on ops not converted by TensorRT. It is also used for
+// the soon-to-be-deprecated enable_mixed_precision_graph_rewrite API.
+// If you will need to modify or remove this API, please kindly contact the
+// TF-TRT owners.
+//
```

Comment:
Sorry, after discussing with reedwm@, we prefer to completely remove this comment, as long as we will have tests for detecting this kind changes.